package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev",
```

Comment:
Remove all scripts from the package.json, developers should use Makefile imstead